talk/owt/sdk/include/cpp/owt/base/globalconfiguration.h

```diff
@@ -55,7 +57,7 @@ struct IcePortRange {
 };
 
 /// Port range settings for different tracks.
-struct IcePortRanges {
+struct OWT_EXPORT IcePortRanges {
   /**
    @brief Port range for audio tracks.
```

Comment:
This one conflicts with previously merged patch to remove ice port ranges. Maybe rebase?

Reply:
Rebased.